docs/adr/adr-006-fraud-service.md

```diff
@@ -211,6 +211,17 @@ func getAll(ctx context.Context, ds datastore.Datastore) ([][]byte, error)
 
 In case if response error will be empty (and not ```datastore.ErrNotFound```), then a BEFP has been already added to storage and the node should be halted.
 
+### Fraud sync
+
+The main purpose of Fraud Sync is to deliver Fraud Proofs to nodes that were started after BEFP appears. Assuming that Full nodes can create the BEFP during reconstruction, Fraud Sync is mainly implemented for Light Nodes:
```

Comment:
```suggestion
The main purpose of FraudSync is to deliver fraud proofs to nodes that were started after a BEFP appears. Since full nodes create the BEFP during reconstruction, FraudSync is mainly implemented for light nodes:
```